src/styles/annotator/bucket-bar.scss

```diff
@@ -4,9 +4,15 @@
 
 // Bucket-bar styles are nested inside `.annotator-frame` to ensure they take
 // precedence over the CSS reset applied to `.annotator-frame`.
-.annotator-frame {
+.annotator-frame.annotator-collapsed {
   .annotator-bucket-bar {
     background: rgba(0, 0, 0, 0.08);
+  }
+}
```

Comment:
This is a quick fix, but not the cleanest ([because of the descendant selectors are slow](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#selectors))